package.json

```diff
@@ -0,0 +1,85 @@
+{
+    "author": {
+        "email": "developer@andculture.com",
+        "name": "andculture",
+        "url": "https://www.andculture.com"
+    },
+    "bugs": {
+        "url": "https://github.com/AndcultureCode/AndcultureCode.JavaScript.Quill/issues"
+    },
+    "dependencies": {
+        "quill-blot-formatter": "1.0.5"
+    },
+    "peerDependencies": {
+        "quill": "1.3.7"
+    },
+    "description": "Common patterns, functions, etc... used when building react applications",
+    "devDependencies": {
+        "@testing-library/jest-dom": "5.5.0",
+        "@testing-library/react": "10.0.4",
+        "@types/faker": "4.1.12",
+        "@types/jest": "25.1.5",
+        "@types/node": "13.11.0",
+        "@types/quill": "2.0.4",
+        "@types/rosie": "0.0.37",
+        "andculturecode-javascript-core": "0.3.6",
+        "andculturecode-javascript-testing": "0.0.1",

```

Comment:
We are up to 0.1.0 for the testing package now - might as well upgrade it while you're in here